plugins/pluginmanagers/lz-n.nix

```diff
@@ -0,0 +1,126 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+let
+  lzPluginType =
+    with helpers.nixvimTypes;
+    let
+      maybeRawStrOrList = either (maybeRaw str) (listOf (maybeRaw str));
+    in
+    submodule {
+      freeformType = attrsOf anything;
+      options = {
+        __unkeyed = mkOption {
+          type = str;
+          description = "The plugin's name (not the module name). This is what is passed to the load(name) function.";
```

Comment:
This description feels a little clunky? Can't quite put my finger on it though...